core/src/test/java/org/apache/calcite/plan/RelOptUtilTest.java

```diff
@@ -405,6 +405,15 @@ private static Frameworks.ConfigBuilder config() {
         relBuilder.literal(true));
   }
 
+  /**
+   * Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-7013">[CALCITE-7013]
+   * Literal should support Character basic type</a>. */
+  @Test void testCharacterLiteral() {
```

Comment:
I am not sure the issue title is suitable; Literals certainly support values of Char type.
There were no suitable constructors taking just a Character value.

Reply:
How about change the title to "Support Character basic type in Literal", maybe is more suiatble. @mihaibudiu

Reply:
It doesn't seem a much better fit.
This is really about creating some utility functions to build RexLiterals directly from Character values.

Reply:
Yes, you are right. Maybe "Support build RexLiterals from Character values" is more in line with this issue？ @mihaibudiu

Reply:
building
yes

Reply:
Done, thank~ @mihaibudiu